core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -341,7 +341,7 @@ public Scope analyze(Node node, Optional<Scope> outerQueryScope)
                 .process(node, Optional.empty());
```

Comment:
"Silence IDE warnings" is not a good explanation. What's the actual logical change being done? Please describe it in those terms and break up the commit into smaller ones if the changes are unrelated (e.g., constrain method visibility, replace lambda expressions with method references, etc)